docs/streams.html

```diff
@@ -46,19 +46,22 @@
         <h2><a id="streams_overview" href="#streams_overview">Overview</a></h2>
 
         <p>
-        Kafka Streams is a client library for processing and analyzing data stored in Kafka and either write the resulting data back to Kafka or send the final output to an external system. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, and simple yet efficient management of application state.
+            Kafka Streams is a client library for processing and analyzing data stored in Kafka and either writes the resulting data back to Kafka or sends the final output to an external system.
```

Comment:
Thinking about this, I think it is Okay to just remove the line as your first suggestion since it is not necessary as part of a high-level description.